### PR TITLE
Correction of the critical formula (renewal)

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: https://github.com/rathena/rathena/issues/7863

* **Server Mode**: renewal

* **Description of Pull Request**: 

Corrections:
* 1 luk = +0.3 critical
* BaseLevel / 10 = +0.1 critical

Before the change on rAthena (character BaseLevel 60):
Capture d’écran (1504)

kRO (character BaseLevel 60):
Capture d’écran (1503)

Now rAthena (character BaseLevel 60):
Capture d’écran (1505)

### CODE REVIEW

File: src/map/status.cpp [hunk 1]
```diff
@@ -2602,7 +2602,12 @@ void status_calc_misc(struct block_list *bl, struct status_data *status, int lev
 	//Critical
 	if( bl->type&battle_config.enable_critical ) {
 		stat = status->cri;
+#ifdef RENEWAL
+		stat += (level / 10); // (every 10 BaseLevel = +0.1 critical)
+		stat += 10 + (status->luk*3); // (every 1 luk = +0.3 critical)
```
Comment (idk-whoami) [inline src/map/status.cpp:2607]: I'm not sure 10/ is missing?

**Review by Lemongrass3110 — APPROVED** [review]

File: src/map/status.cpp (hunk 1 above)
Comment (Atemo) [reply src/map/status.cpp:2607]: What do you mean, any example?

Comment (idk-whoami) [reply src/map/status.cpp:2607]: I saw the original one like this.
```
stat += 10 + (status->luk*10/3); // (every 1 luk = +0.3 critical)

Comment (Atemo) [reply src/map/status.cpp:2607]: Ha no this change is intended.

**Pull request merged**